config/routes.rb

```diff
@@ -24,6 +24,7 @@
     resources :action_items, only: [:index]
     resources :drafts, only: [:index]
     resources :proposals, only: [:create]
+    resources :skills, only: [:index, :new, :create, :edit, :update]
```

Comment:
Do we want to allow admins to delete skills? I guess they can edit if there is a typo